apps/meteor/app/api/server/v1/channels.ts

```diff
@@ -806,7 +806,7 @@ API.v1.addRoute(
 
 API.v1.addRoute(
 	'channels.images',
-	{ authRequired: true, validateParams: isChannelsImagesProps },
+	{ authRequired: true, validateParams: isRoomsImagesProps, deprecationVersion: '7.0.0' },
```

Comment:
looks like the `addRoute` API doesn't support the "info" param from `apiDeprecationLogger.endpoint` so there is no way to recommend a new endpoint since this one is deprecated.. 

I can see two options in this case:

- We add the support to this API to pass additional info, it could be something like:
```ts
API.v1.addRoute(
	'channels.images',
	{ authRequired: true, validateParams: isRoomsImagesProps, deprecationInfo: 'Use /api/v1/rooms.images instead.', deprecationVersion: '7.0.0' },
     // ...
);
```
- Or we don't use this API and call `apiDeprecationLogger.endpoint` direct from the endpoint's body:
```ts

API.v1.addRoute(
	'channels.images',
	{ authRequired: true, validateParams: isRoomsImagesProps },
	{
		async get() {
			// ...
			apiDeprecationLogger.endpoint(this.request.route, '7.0.0', this.response, 'Use /v1/rooms.images instead.');
			// ...
		}
	}
);
```

what do you think? I think either are fine.

Reply:
Option 2 was fine, went with that one 😬